src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -17,6 +17,21 @@ export interface AmalgamationResourceIF {
   }
 }
 
+/** Continuation in resource interface. */
+export interface ContinuationInResourceIF {
+  entityType: CorpTypeCd
+  displayName: string
+  steps: Array<StepIF>
+  filingData: Array<FilingDataIF>
+  peopleAndRoles: PeopleAndRolesResourceIF
+  reviewAndConfirm: {
+    completingPartyStatement: CompletingPartyStatementIF
+  }
+  shareClasses?: {
+    countMinimum: number
```

Comment:
I think this isn't used anywhere, actually. Double-check and remove if not needed.